shared-libs/rules-engine/src/rules-state-store.js

```diff
@@ -86,11 +88,11 @@ const self = {
     }
 
     const now = Date.now();
-    const { calculatedAt, isDirty } = state.contactState[contactId];
-    return !calculatedAt ||
+    const { calculatedAt, expireAt, isDirty } = state.contactState[contactId];
+    return !expireAt ||
       isDirty ||
-    /* user rewinded their clock */ calculatedAt > now ||
-    /* isExpired */ calculatedAt < now - EXPIRE_CALCULATION_AFTER_MS;
+    /* user rewound their clock */ calculatedAt > now ||
```

Comment:
https://en.wiktionary.org/wiki/rewinded

Reply:
Both look bad, the analogy is dated, and it's not the users fault. How does one wind up a smartphone clock anyway? It'd be better to say "system clock changed"

Reply:
https://wikidiff.com/rewinded/rewound
My IDE was complaining it was misspelled. All sources said that rewinded (chrome complains it's misspelled as I typed it) was 'nonstandard'. 
But if the English speakers say so ... I'll change it back :)